Classes/KIFUITestActor.h

```diff
@@ -50,6 +50,22 @@ typedef NS_ENUM(NSUInteger, KIFStepperDirection) {
 	KIFStepperDirectionDecrement
 };
 
+/*!
+ @enum KIFStepperPullToRefreshTiming
+ @discussion The approximate time in which the pull to refresh travels downward.
+ @abstract Timing in which to perform the pull down before release.
+ @constant KIFPullToRefreshInAboutAHalfSecond about half a second
+ @constant KIFPullToRefreshInAboutOneSecond about one second
+ @constant KIFPullToRefreshInAboutTwoSeconds about two seconds
+ @constant KIFPullToRefreshInAboutThreeSeconds about three seconds.
+ */
+typedef NS_ENUM(NSUInteger, KIFStepperPullToRefreshTiming) {
```

Comment:
I doubt you meant to call this `KIFStepperPullToRefreshTiming`. Looks like a copy/paste error from the enum above :-)